webapp/views/results-modal/script.js

```diff
@@ -1,16 +1,156 @@
-/* global $ */
+/* global $, get, deleteMethod */
 
-// eslint-disable-next-line no-unused-vars
-function display() {
-  const value = $('#results-select')[0].value;
-  if (value != '') {
-    $('#results-iframe').attr('src', '/output/' + value);
-  } else {
-    $('#results-iframe').attr('src', '');
+/**
+ * result modal class
+ */
+class ResultModal {
+  constructor(data = [], resultToDelete = '', currentView = 'ALL') {
```

Comment:
Where do you use the constructor arguments?

Reply:
Also, "data"... data can be everything and anything. "x" would be just as good. Please keep in mind that variable names should help the reader understand what's going on.